staging/src/k8s.io/cloud-provider/options/options.go

```diff
@@ -42,9 +42,6 @@ import (
 	cmoptions "k8s.io/controller-manager/options"
 	"k8s.io/controller-manager/pkg/clientbuilder"
 	netutils "k8s.io/utils/net"
-
-	// add the related feature gates
```

Comment:
maybe they want to keep it for later? better ask Andrew and Walter before removing

Reply:
Did we get a decision on this?

Reply:
This is what https://github.com/kubernetes/kubernetes/pull/115204/commits/8360f9c0302f6edfdd5b13bbe69f7f283a1d4742#r1107217918 talks about, right? So I understand we the decision is to remove the commit from this PR